Sources/Core/AWSClientRuntime/Sources/AWSClientRuntime/Config/AWSDefaultClientConfiguration.swift

```diff
@@ -31,9 +31,19 @@ public protocol AWSDefaultClientConfiguration {
 
```

Comment:
Adds back the lost `maxAttempts` option back to config. Welcome back to our poor option that finally found its way back home.